lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
I agree the name would be more explanatory, but I used the same as in `line-comment-position` for consistency, so I'm hesitant to use a different it here. :/